src/github.com/couchbase/sync_gateway/rest/changes_api.go

```diff
@@ -93,7 +93,7 @@ func (h *handler) handleChanges() error {
 		if channelsParam != "" {
 			channelsArray = strings.Split(channelsParam, ",")
 		}
-		options.HeartbeatMs = getRestrictedIntQuery(h.rq.URL.Query(), "heartbeat", kDefaultHeartbeatMS, kMinHeartbeatMS, h.server.config.MaxHeartbeat * 1000, true)
+		options.HeartbeatMs = getRestrictedIntQuery(h.rq.URL.Query(), "heartbeat", kDefaultHeartbeatMS, kMinHeartbeatMS, h.server.config.MaxHeartbeat*1000, true)
```

Comment:
I think this would be easier to read as:

```
getRestrictedIntQuery(
                         h.rq.URL.Query(),
                         ...,
)
```